### PR TITLE
update title of Chinese version

### PR DESCRIPTION
**Pull request by WayneWang12**
As mentioned in https://github.com/reactivemanifesto/reactivemanifesto/pull/115, update the title of manifesto to "反应式宣言".

### CODE REVIEW

**Comment by hepin1989** [comment]
@WayneWang12 So does the svg

File: public/images/reactive-traits-zh-cn.svg [hunk 1]
```diff
@@ -27,9 +27,9 @@
 		<line class="st0" x1="788.8" y1="193" x2="914.8" y2="193"/>
 	</g>
 	<g>
-		<text transform="matrix(1 0 0 1 5.7999 148.965)" class="st1 st2 st3">FORM</text>
-		<text transform="matrix(1 0 0 1 5.7999 238.965)" class="st1 st2 st3">MEANS</text>
-		<text transform="matrix(1 0 0 1 5.7999 68.965)" class="st1 st2 st3">VALUE</text>
+		<text transform="matrix(1 0 0 1 5.7999 148.965)" class="st1 st2 st3">手段</text>
+		<text transform="matrix(1 0 0 1 5.7999 238.965)" class="st1 st2 st3">形式</text>
+		<text transform="matrix(1 0 0 1 5.7999 68.965)" class="st1 st2 st3">价值</text>
```
Comment (hepin1989) [inline public/images/reactive-traits-zh-cn.svg]: Just keep it for now? We need another updating here.

**Comment by hepin1989** [comment]
@WayneWang12 Thanks for your hard work.

**Comment by hepin1989** [comment]
@rasummer it's Ready to merge I think.

**Comment by hepin1989** [comment]
@rasummer When will the website be updated?

**Comment by rasummer** [comment]
@jroper when you get a chance, could you deploy the site. thanks

**Comment by jroper** [comment]
Deployed.

**Pull request merged**